apps/explorer/pages/details/[id].vue

```diff
@@ -19,13 +20,18 @@
 <script lang="ts" setup>
 import { useRoute } from 'vue-router'
 import { Soul } from '~~/types/soul'
+import { SoulProperty } from '~~/types/soul-property'
 
 // const souls = useSouls()
 useGqlCors({ credentials: 'same-origin' })
 const route = useRoute()
 
-const { data, error } = await useAsyncGql('sniList', { sniId: '1' })
-const detail = data.value.snis.filter((detail) => detail.id === route.params.id)
+const { data, error } = await useAsyncGql('sniList', {
+  sniId: route.params.id.toString()
+})
+const detail = data.value.sni
+const properties = data.value.sni.properties
+delete detail.properties
```

Comment:
Why you need to delete a field?

Reply:
Because this is still the request for a single Detail type (gql  - sni(id).
I've tried to create a query just for that, but that doesn't work, So I need to take the properties and then remove them for not display them in the SNIDetails component

Reply:
We'll improve data structure later